MediaManager/Plugin.MediaManager.Abstractions/IMediaManager.cs

```diff
@@ -63,11 +63,6 @@ public interface IMediaManager : IPlaybackManager
         event MediaFileFailedEventHandler MediaFileFailed;
 
         /// <summary>
-        /// Creates new MediaFile object, adds it to the queue and starts playing
-        /// </summary>
-        Task Play(string url, MediaFileType fileType);

```

Comment:
This breaks stuff. And it doesn't make things easier for users. When we find a way to detect if a file is video or audio, we should add `Task Play(string url)` as well.